pkg/networking/cilium/reconciler/marker.go

```diff
@@ -0,0 +1,29 @@
+package reconciler
+
+import (
+	"context"
+
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+)
+
+// EmbeddedCiliumInstalledAnnotation indicates a cluster has previously been observed to have
+// EKS-A Embedded Cilium installed irrespective of whether its still installed.
+const EmbeddedCiliumInstalledAnnotation = "anywhere.eks.amazonaws.com/embedded-cilium"
```

Comment:
Curious why adding the word Embedded as opposed to just "EKS-A Cilium"?
I suspect the moment we introduce this term, we are all gonna start using it.

Reply:
The 'embedded' term has been floated in a few meetings. I can strip it out entirely.